docs/reference/release-notes/8.10.4.asciidoc

```diff
@@ -27,6 +27,8 @@ running version 8.10.4.
 
 Also see <<breaking-changes-8.10,Breaking changes in 8.10>>.
 
+include::8.8.0.asciidoc[tag=repo-analysis-missing-timeout]
+
```

Comment:
```suggestion
include::8.8.0.asciidoc[tag=repo-analysis-missing-timeout]

Also see <<breaking-changes-8.10,Breaking changes in 8.10>>.

```
Otherwise, the "Also see..." is between the two issues.